client/src/views/Settings/SettingsMenu.vue

```diff
@@ -1,18 +1,54 @@
 <template>
-  <ul>
-    <li v-for="(item, index) in routeItems" :key='index'>
-      <router-link :to="item.to">{{item.title}}</router-link>
-    </li>
-  </ul>
+  <div>
+    <q-card class="my-card q-ma-sm">
+      <q-splitter
+        v-model="splitterModel"
+        :[attrPosition]="true"
```

Comment:
You could simplify here, no need to listen resize instead you could you quasar break-points($q.screen.xs).
```suggestion
       :horizontal="horizontalAttrPosition"
       :vertical="!horizontalAttrPosition"
       
       computed () {
         horizontalAttrPosition () {
           return this.$q.screen.xs
       }
```